test/cid-version-agnostic.js

```diff
@@ -43,99 +44,98 @@ describe('CID version agnostic', () => {
   })
 
   after(function () {
-    this.timeout(30 * 1000)
-    return Promise.all(Object.values(daemons).map(stopDaemon))
+    return Promise.all(Object.values(daemons).map((daemon) => daemon.stop()))
   })
 
-  it('should add v0 and cat v1 (go0 -> go0)', async () => {
+  it('should add v0 and cat v1 (go0 -> go0)', async function () {
```

Comment:
Why did this change from an arrow function?

Reply:
I added a custom `this.timeout()` to these tests. Then I removed it and I forgot to change the function back to an arrow function.